### DISCUSSION TITLE
Migrate to OpenTofu from Terraform

### DISCUSSION THREAD

**Discussion by caniko** [mysticaltech/terraform-hcloud-kube-hetzner#1166] [Ideas]
OpenTofu just had its [first stable release](https://opentofu.org/blog/opentofu-is-going-ga/), I think we should at least start discussions for migrating over there.

Those that are familiar with every detail of the project should probably take a look at the [issue tracker](https://github.com/opentofu/opentofu/issues) and project to see if there are any detrimental bugs or missing features.

**Comment by caniko** [comment]
[Add support for for_each in the import block](https://github.com/opentofu/opentofu/issues/1033) might be an issue?

**Comment by mysticaltech** [comment]
Already done, opentofu is supported now.
